app/controllers/apitome/docs_controller.rb

```diff
@@ -80,14 +80,19 @@ def rendered_markdown(string)
     end
 
     def formatted_body(body, type)
+      return body if body == " "
+      return body if body == "[binary data]" # see https://github.com/zipmark/rspec_api_documentation/blob/560c3bdc7bd5581e7c223334390221ecfc910be8/lib/rspec_api_documentation/client_base.rb#L88-L96
```

Comment:
refactor  and 🐞 fix

Reply:
the `"[binary data"]` change was related in our app to changing from -    rack (2.0.8) to rack (2.1.1)

Reply:
related to https://github.com/zipmark/rspec_api_documentation/pull/458 https://github.com/zipmark/rspec_api_documentation/issues/456